tests/copier_data/data2.yaml

```
@@ -7,6 +7,8 @@ install_claude_cli: true
 ssh_port_number: 54321
 
 
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (4 > 2)

(empty-lines)